doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1,3 +1,5 @@
+:orphan:
+
```

Comment:
The known issues subpage needs to be removed from the "releases_and_maturity.rst" to hide the Known issue page for the devtag. 
https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-14960/nrf/releases_and_maturity.html#

Reply:
done